src/vivarium_testing_utils/automated_validation/data_loader.py

```diff
@@ -34,45 +36,53 @@ def __init__(self, results_dir: str, cache_size_mb: int = 1000):
             DataSource.CUSTOM: self._load_custom,
         }
         self.metadata = LayeredConfigTree()
-        self.artifact = None  # Just stubbing this out for now
+        self.artifact = self._load_artifact(self.sim_output_dir)
```

Comment:
I think we may have situations where we would want multiple artifacts to be in scope for a single `ValidationContext`. We currently store the data for different countries in separate artifacts, but I believe the RT regularly creates notebooks with the results for all locations. This suggests to me that there are a few options:
- a notebook could have a separate `ValidationContext` for each location
- a `ValidationContext` could have a separate `DataLoader` for each location
- a `DataLoader` could contain multiple `Artifacts`